src/styles/stars.scss

```diff
@@ -0,0 +1,28 @@
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  height: 16px;
+  margin-bottom: 24px;
+
+  &-stars {
+    display: flex;
+
+    &-star {
+      height: 16px;
+      width: 16px;
+      background-image: url(../images/star.svg);
+      background-repeat: no-repeat;
+      background-position: center;
+
+    }
+
+    &-star:not(:last-child) {
+      margin-right: 4px;
+    }
+
+    &-star:nth-child(-n + 4) {
```

Comment:
apply styles for all cases (when you have 1, 2, 3, 4, 5 active stars. The same as you did earlier)